src/patternfly/components/Drawer/drawer.scss

```diff
@@ -231,6 +233,13 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   overflow: auto;
 }
 
+.pf-c-drawer__panel-main {
+  @extend .pf-c-drawer__panel;
```

Comment:
This may work in this context, but we usually do not use `@extend` as it can cause unpredictable issues, particularly in a large project. It will extend any instance of `.pf-c-drawer__panel` anywhere in the stylesheet, including compound selectors. The first part of [this article](https://webinista.com/updates/dont-use-extend-sass/) covers it pretty well.

A better alternative could be to use a placeholder or `@mixin` for shared blocks like this, however in this instance I would suggest just adding `.pf-c-drawer__panel-main` to any `.pf-c-drawer__panel` styles that need to be shared.

Reply:
Here's a good example of styles from `.pf-c-drawer__panel` that apply to `.pf-c-drawer__panel-main` now, and you don't want any of them.

<img width="376" alt="Screen Shot 2021-01-11 at 10 54 48 AM" src="https://user-images.githubusercontent.com/35148959/104213300-beec8380-53fb-11eb-8a3e-a3af2a85fbd5.png">

The only rule in there that you want on `__panel-main` is `overflow: auto;` but that will come from this block when you add `__panel-main` to it:

https://github.com/patternfly/patternfly/blob/506ec62cc4b7c824f42ad6f14cd37a0a25538c68/src/patternfly/components/Drawer/drawer.scss#L225-L232

Reply:
@mcoker I see! I had never expected using `@extend` would cause such big problems. This is a great reference and thank you for pointing that out. I will fix it!